[BUG] 解密数据时没有校验iv导致panic

**描述**
go的aes库在iv和key长度不匹配的时候会直接panic

**如何复现**
```golang
func TestGetCipherText_BadIV(t *testing.T) {
	keyData := base64.StdEncoding.EncodeToString([]byte("1234567890123456"))
	badData := base64.StdEncoding.EncodeToString([]byte("1"))
	_, _ = GetCipherText(keyData, badData, badData)
}
```
**关联日志信息**
```
panic: cipher.NewCBCDecrypter: IV length must equal block size [recovered]
	panic: cipher.NewCBCDecrypter: IV length must equal block size

goroutine 6 [running]:
testing.tRunner.func1.2({0x4f79e0, 0x566690})
	C:/Program Files/Go/src/testing/testing.go:1209 +0x24e
testing.tRunner.func1()
	C:/Program Files/Go/src/testing/testing.go:1212 +0x218
panic({0x4f79e0, 0x566690})
	C:/Program Files/Go/src/runtime/panic.go:1038 +0x215
crypto/cipher.NewCBCDecrypter({0x569ef8, 0xc0000770b0}, {0xc000010cd3, 0x0, 0x3})
	C:/Program Files/Go/src/crypto/cipher/cbc.go:107 +0xcc
github.com/silenceper/wechat/v2/miniprogram/encryptor.GetCipherText({0xc000014708, 0xc000055f3f}, {0xc000010ccc, 0x4}, {0xc000010ccc, 0x4})
	F:/workspace/silenceper/wechat/miniprogram/encryptor/encryptor.go:97 +0x139
github.com/silenceper/wechat/v2/miniprogram/encryptor.TestGetCipherText(0x0)
	F:/workspace/silenceper/wechat/miniprogram/encryptor/encryptor_test.go:11 +0x95
```

**使用的版本**
 - SDK版本: v2.0.6

--- a/miniprogram/encryptor/encryptor.go
+++ b/miniprogram/encryptor/encryptor.go
@@ -6,6 +6,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"errors"
+	"fmt"
 
 	"github.com/silenceper/wechat/v2/miniprogram/context"
 )
@@ -89,6 +90,9 @@ func GetCipherText(sessionKey, encryptedData, iv string) ([]byte, error) {
 	ivBytes, err := base64.StdEncoding.DecodeString(iv)
 	if err != nil {
 		return nil, err
+	}
+	if len(ivBytes) != aes.BlockSize {
+		return nil, fmt.Errorf("bad iv length %d", len(ivBytes))
 	}
 	block, err := aes.NewCipher(aesKey)
 	if err != nil {

--- a/miniprogram/encryptor/encryptor_test.go
+++ b/miniprogram/encryptor/encryptor_test.go
@@ -1,0 +1,15 @@
+package encryptor
+
+import (
+	"encoding/base64"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetCipherText_BadIV(t *testing.T) {
+	keyData := base64.StdEncoding.EncodeToString([]byte("1234567890123456"))
+	badData := base64.StdEncoding.EncodeToString([]byte("1"))
+	_, err := GetCipherText(keyData, badData, badData)
+	assert.Error(t, err)
+}

